demo/admin/src/predefinedPage/PredefinedPage.tsx

```diff
@@ -74,4 +74,8 @@ export const PredefinedPage: DocumentInterface<Pick<GQLPredefinedPage, "type">,
         }
     },
     anchors: () => [],
+    resolveDependencyRoute: () => {
+        // TODO
+        return "";
+    },
```

Comment:
@johnnyomair I'm not sure how I can generate the URL here (or if it's even necessary). Can you give me a hint?